BankingKataSpecs/AccountSpecs/Features/CreateAccount.feature

```diff
@@ -0,0 +1,19 @@
+﻿Feature: Create account
+	In order to us my account
+	As a foo user
+	I want to be able to create an account.
+
+@mytag
+Scenario: New account has zero cash.
+	Given I want to create an account
+	When I create a new account
+	Then the account should have 0 cash
+
+Scenario: New account can't take null cash.
+	Given I want to create an account
+	And I have null cash
+	Then Creating an account should throw an argument null exception
```

Comment:
This test feels very dev-oriented, due to talking about `null` and exceptions. I'm not sure whether there would be a more user focused way of writing it (`Given I have an unknown amount of cash Then creating an account should give an error`?), or whether the test fits at all. Maybe BDD doesn't lend itself well to testing failure conditions?